traitlets/config/configurable.py

```diff
@@ -221,17 +221,20 @@ class defaults.
         base_classes = ','.join(p.__name__ for p in cls.__bases__)
         final_help.append(u'%s(%s) options' % (cls.__name__, base_classes))
         final_help.append(len(final_help[0])*u'-')
-        for k, v in sorted(cls.class_traits(config=True).items()):
+        for k, v in sorted(cls.class_own_traits(config=True).items()):
```

Comment:
It seems like this should still be `.class_traits`, since it should include all configurables on the class, not just those it didn't inherit.